### DISCUSSION TITLE
BFE多IDC多集群部署场景

### DISCUSSION THREAD

**Discussion by beebol** [bfenetworks/bfe#1206] [General]
多BFE集群场景下，增加一个BFE集群，需要将所有子集群调度负载中增加BFE集群，否则新增加的BFE集群中的conf-agent无法正常拉取到配置，会报如下错误：[2024/12/26 15:54:25 CST] [WARN] [github.com/bfenetworks/api-server/endpoints/middleware.Record:75] [1735199665109049205_539]cost_ms[307] method[GET] pattern[/configs/gslb_data/gslb] path[/inner-api/v1/configs/gslb_data/gslb] client_ip[] status_code[422] ret_msg[Param Illegal: BFECluster BFE-xxx Not Exist] err_detail[]]

在实际场景中，肯定会涉及到有部分业务集群是不需要部署在所有的BFE集群的，这块应该如何配置？

**Comment by ohscartao** [comment]
1，开源版本在BFE增加BFE集群后，确实需要给业务Cluster增加调度配置。
2，实际场景中，存在部分业务集群仅需部署在某些BFE集群，不需要全部部署。这种情况下，一般通过4层负载均衡的VIP来隔离不同的业务：每个业务分配自己的VIP，每个VIP绑定对应的BFE集群。
